renv.lock

```diff
@@ -2546,6 +2561,18 @@
         "zoo"
       ]
     },
+    "lobstr": {
+      "Package": "lobstr",
+      "Version": "1.1.1",
+      "Source": "Repository",
+      "Repository": "RSPM",
```

Comment:
This confuses me a bit. Presumably you passing in `https://packagemanager.posit.co/cran/2021-06-29` as repo when adding butcher, as per Tom's issue.  But what we end up with is just "RSPM". We seem to have lost the "https://packagemanager.posit.co/cran/2021-06-29" url somehow. So I suspect this won't build on another system that doesn't have the build cached.

Naively, I would have expected a new "RSPM" entry in the "Repositories" section of renv.lock, but this doesn't seem to be in this commit?

Reply:
Great spot that there isn't an entry for RSPM in "Repositories" in renv.lock (RSPM = RStudio Package Manager a.k.a. Posit Package Manager). This turns out to be ok because CRAN is listed in *renv.lock* and `renv::restore()` falls back to that, so it will restore when **lobstr** not in the cache.

Here's a screenshot of the output from my test example, which is [here](https://github.com/remlapmot/test-renv/blob/32b24bd4d7e53d2b9fa9204b506566a2584df941/main-01.Rout#L360-L379)
<img width="551" alt="2023-02-26 08_55_21-AsPowerBar" src="https://user-images.githubusercontent.com/3777473/221401221-7095d195-a830-4818-b15e-d17a9680c7db.png">